src/test/java/org/codehaus/plexus/util/xml/Xpp3DomUtilsTest.java

```diff
@@ -60,6 +60,41 @@ public void testCombineId()
         assertEquals( "right", p2.getChild( "value" ).getInputLocation() );
     }
 
+    @Test
+    public void testCombineKeys()
+        throws Exception
+    {
+        String lhs = "<props>" + "<property combine.keys='name'><name>LHS-ONLY</name><value>LHS</value></property>"
```

Comment:
is this ok? according to the readme in https://github.com/atteo/xml-combiner, the value of combine.keys are attributes names, and here are tag names.

Reply:
Hmm, I see what you mean. In case of Maven I am missing the ability to identity the key of complex objects.
But there's indeed something else wrong: `combine.keys` shõuld not be on the element itself, but the wrapping element. And with that I understand why it is for attributes. I need to have another look at it.

Great catch!

Reply:
https://github.com/codehaus-plexus/plexus-utils/commit/e1304ddeb9aef72ac27a1b7f7e58908cc7cee286